composer.json

```diff
@@ -20,13 +20,16 @@
         }
     ],
     "require": {
-        "php": "^7.3",
-        "illuminate/http": "^6.0|^7.0|^8.0"
+        "php": "^7.3|^8.0",
+        "illuminate/http": "^6.0|^7.0|^8.0",
+        "laravel/framework": "6.*",
+        "orchestra/testbench": "4.*",
```

Comment:
This line is included in the matrix build as well, so I think we'd better not lock it on 4.*.